tests/core/test_polydata.py

```diff
@@ -286,6 +286,13 @@ def test_ray_trace(sphere):
     assert np.any(ind)
 
 
+def test_ray_trace_origin():
+    # Regression test for issue #5372
```

Comment:
It is more convenient to describe the URL directly so that you can go to the issue immediately.
```suggestion
    # https://github.com/pyvista/pyvista/issues/5372
```